All EIPIP Meetings/Meeting 028.md

```diff
@@ -0,0 +1,551 @@
+
+**Edson Allyon**
+Is that really an issue?
+
+**Hudson Jameson**
+For people who want to review at the last minute. There are some people who look at the RSS feed for Last Call. And that's when they start to look at an EIP.
+
+**Edson Allyon**
+I think the solution is just having RSS speak for Review instead of one for Last Call.
+
+**Micah**
+Do we know if there's more than one person that uses the RSS feed?
+
+**Hudson Jameson**
+Yes, I know two. So that could mean that could mean that there are more than two. So I don't know.
+
+**Micah**
+Yeah. I was just wondering if we're optimizing for the right crowd?
+
+**Hudson Jameson**
+Well, yeah, I get I guess you're right.
+
+**Micah**
+I honestly don't know. Maybe there's thousands of people who subscribe to the RSS feed. I know one.
+
+**Hudson Jameson**
+It's just the statuses is the thing. The status does indicate people rushing to review it, and then they start to disagree with it, even though they're not looking because who's looking for a security vulnerability in the last 14 days? I don't think many people are.
+
+**Micah**
+So I think the real problem here is this. In my experience, the vast majority of EIPs get a grand total of one to two people looking at it right up until a bunch of people need to implement it. And then you have a thousand people looking at it. And so when a thousand people start looking at it in the Last Call phase, then you get a lot of feedback. And if you somehow get a lot of people looking at in the Review phase, like with 1559, then you get a lot of feedback in the Review phase. Actually, 1559, I got a lot of feedback in the Draft phase. So I feel like we have these statuses that are meant to give people information, but I don't feel like they're particularly functional in achieving that goal. People are not looking at them.
+
+**Hudson Jameson**
+I would lean toward this to help with the fact that to basically stick with somewhat of the status quo, but to try to also push best practices would be to keep Last Call's definition as is, pretty much, because it doesn't include all the security vulnerabilities right now and move it to DevNet phase. And then public testnet doesn't include an EIP status beyond... but then that leaves public testnets to not have an EIP status. That's the problem, I guess. Yeah.
+
+**Pooja Ranjan**
+One more thing I would like to add here is generally, the definition of Last Call for that is [unclear](https://youtu.be/fKQVb-s0Tzs?t=1128). So one correction there would be a minimum of 14 days would be good. But you know, if they want to keep it in Last Call for over a month, because when it is on testnet, they believe it is like [unclear](https://youtu.be/fKQVb-s0Tzs?t=1139). So I am not very sure that we should put the Last Call status in DevNet phase.
```

Comment:
```suggestion
One more thing I would like to add here is generally, the definition of Last Call suggests it stays there for 14 days. So one correction there would be "a minimum of 14 days" would be good. But you know, if we want to keep it in Last Call for over a month, because when it is on testnet, I believe it is like four weeks at least. So I am not very sure that we should put the Last Call status in the DevNet phase.
```